Tests/aweXpect.Reflection.Tests/ThatMethods.AreGeneric.Tests.cs

```diff
@@ -0,0 +1,41 @@
+using aweXpect.Reflection.Collections;
+
+namespace aweXpect.Reflection.Tests;
+
+public sealed partial class ThatMethods
+{
+	public sealed class AreGeneric
+	{
+		public sealed class Tests
```

Comment:
Also add tests for the negated case using `DoesNotComplyWith` (see [ThatFields.ArePublic.NegatedTests](https://github.com/aweXpect/aweXpect.Reflection/blob/7ca54066e02e76194e0e78a2d1dc751c13b8009e/Tests/aweXpect.Reflection.Tests/ThatFields.ArePublic.Tests.cs#L45) for an example).